src/test/lib/repeat_test.ts

```diff
@@ -145,6 +145,19 @@ suite('repeat', () => {
       assert.equal(container.innerHTML, `<li>item: 2</li>`);
     });
 
+    test('can rerender repeated items with no changes', () => {
+      const t = (items: number[]) =>
+          html`${repeat(items, (i) => i, (i: number) => html`
+          <li>item: ${i}</li>`)}`;
+
+      render(t([666, 666]), container);
+      render(t([666, 666]), container);
+
+      assert.equal(
+          container.innerHTML,
+          `<li>item: 666</li><li>item: 666</li>`);
```

Comment:
This should be just `<li>item: 666</li>`.

Reply:
It should render two items right?

Reply:
It should render two items right?

Reply:
Nah, it'll find the same part (with id `666`), and update it. See https://github.com/PolymerLabs/lit-html/pull/188/files/cab9000cc4d6a777a47122a3d829c777cd3810a5#diff-df279e375bd22a90dee8ee7e0ff80d13R125.